ProcessLib/RichardsMechanics/RichardsMechanicsFEM-impl.h

```diff
@@ -194,9 +219,8 @@ void RichardsMechanicsLocalAssembler<
         auto const mu = _process_data.flow_material->getFluidViscosity(
             -p_cap_ip, temperature);
 
-        double const K_intrinsic =
-            _process_data.intrinsic_permeability(t, x_position)[0];
-        double const K_over_mu = K_intrinsic / mu;
```

Comment:
Why not keep `double const K_over_mu = k_rel / mu * K_intrinsic;` This expression is used twice below. Maybe pre-calculating the value can save some time?